src/ring_theory/derivation.lean

```diff
@@ -199,11 +200,11 @@ instance : has_bracket (derivation R A A) (derivation R A A) := ⟨derivation.co
 lemma commutator_apply : ⁅D1, D2⁆ a = D1 (D2 a) - D2 (D1 a) := rfl
 
 instance : lie_ring (derivation R A A) :=
-{ add_lie     := λ d e f, by { ext a, simp only [commutator_apply, add_apply, map_add], ring, },
-  lie_add     := λ d e f, by { ext a, simp only [commutator_apply, add_apply, map_add], ring, },
-  lie_self    := λ d, by { ext a, simp only [commutator_apply, add_apply, map_add], ring_nf, },
+{ add_lie := λ d e f, by ext; simp only [commutator_apply, coe_add, map_add, pi.add_apply]; ring,
+  lie_add := λ d e f, by ext; simp only [commutator_apply, coe_add, map_add, pi.add_apply]; ring,
+  lie_self := λ d, by { ext a, simp only [commutator_apply, coe_add, map_add], ring_nf, },
```

Comment:
Mathlib style prefers `,` not `;` when possible, so you should probably revert this change (unless the proof has really changed here)

Reply:
The point is that I added `pi.add_apply` because I changed the `simp` lemmas and that was the only way to keep one-line proofs... Is it important to follow that style rule?

Reply:
It's more important than keeping proofs to one line